.bit/responses/1.1-Week 1 Step 1.md

```diff
@@ -11,20 +11,15 @@ Week 1 Step 1 ⬤◯◯◯◯◯◯◯◯ | 🕐 Estimated completion: 5-20 minu
 ## GitHub
 *This week, you will be going through steps to set up tools needed to be successful in this camp. If you are already familiar with some, feel free to skip to the end and complete the task to move on.*
 
-### ✅  Task:
+### ✅  Tasks:
+- [ ]  ***1:*** Create a new branch named `test`.
```

Comment:
Awesome!! Would just suggest adding links to the headers using # 

Example: `[1.](#a-header)` Create a new branch named `test`.